app/src/main/java/org/mozilla/fenix/settings/deletebrowsingdata/DeleteBrowsingDataFragment.kt

```diff
@@ -147,7 +147,7 @@ class DeleteBrowsingDataFragment : Fragment(R.layout.fragment_delete_browsing_da
         if (popAfter) viewLifecycleOwner.lifecycleScope.launch(
             Dispatchers.Main
         ) {
-            returnToDeletionOrigin()
+            findNavController().popBackStack()
```

Comment:
A little worried about this interaction, I think maybe the `returnToDeletionOrigin` also pops off the browser fragment in case the user closes all tabs? What do you think?